### ISSUE TITLE
Ansible Tower AWS Workshop | Update key creation with mode to avoid Ansible warning

### ISSUE THREAD

**Issue by dmc5179** [RedHatGov/redhatgov.workshops#114]
When provisioning the workshop the defaults for file creation have changed resulting in a warning:

```
TASK [admin_server_prep : copy template to working directory] ************************************************************************************
[WARNING]: File '/redhatgov.workshops/ansible_tower_aws/.redhatgov/aws_credentials_vault.yml.pre' created with
default permissions '600'. The previous default was '666'. Specify 'mode' to avoid this warning.
changed: [localhost]

TASK [admin_server_prep : copy password file to working directory] *******************************************************************************
[WARNING]: File '/redhatgov.workshops/ansible_tower_aws/.redhatgov/workshop-password' created with default
permissions '600'. The previous default was '666'. Specify 'mode' to avoid this warning.
changed: [localhost]


TASK [admin_server_prep : stage group_vars/all.yml for remote host] ******************************************************************************
[WARNING]: File '/redhatgov.workshops/ansible_tower_aws/.redhatgov/all.yml' created with default permissions
'600'. The previous default was '666'. Specify 'mode' to avoid this warning.
```

**Comment by dmc5179** [comment]
https://github.com/RedHatGov/redhatgov.workshops/pull/119

**Comment by ajacocks** [comment]
Closed by #119
